umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -109,8 +109,13 @@
                     ;; The RelatedUrls field get reshuffled during the conversions,
                     ;; so we compare RelatedUrls as a set.
                     expected (update expected :RelatedUrls set)
-                    actual (update actual :RelatedUrls set)]]
-
+                    actual (update actual :RelatedUrls set)
```

Comment:
I created a function in the PR for CMR-3849 to do this, so move this code into that function when you merge with me.